maxwell_slice/sphere_scat.py

```diff
@@ -17,18 +17,40 @@ def sphere_scat():
         shutil.copyfile(f'{thisdir}/../scratchspace/SphereScat/controls.dat', f'{tmpdir}/controls.dat')
         shutil.copyfile(f'{thisdir}/../scratchspace/SphereScat/spinsimp.dat', f'{tmpdir}/spinsimp.dat')
 
+        nnx = input("Enter nnx: ")
```

Comment:
This will work, but it will be more useful if nnx, nny, nnz are input arguments to the sphere_scat function. And you can put defaults in like this:

```
def sphere_scat(nnx=64, nny=64, nnz=64):
    ...
```

Then you can pass those in from wherever we are calling it from.

Reply:
Makes sense! I'll be sure to change that

Reply:
I made that change! I set the defaults to 20. Reading about jinja stuff now